websockify.js

```diff
@@ -80,31 +81,11 @@ var http_request = function (request, response) {
 		return http_error(response, 403, "403 Permission Denied");
 	}
 
-	var uri = url.parse(request.url).pathname
-		, filename = path.join(argv.web, uri);
-	
-	fs.exists(filename, function(exists) {
-		if(!exists) {
-			return http_error(response, 404, "404 Not Found");
-		}
-
-		if (fs.statSync(filename).isDirectory()) {
-			filename += '/index.html';
-		}
-
-		fs.readFile(filename, "binary", function(err, file) {
-			if(err) {
-				return http_error(response, 500, err);
-			}
-
-			response.writeHead(200);
-			response.write(file, "binary");
```

Comment:
@maximegris this causes:

```
Refused to execute script from 'data.min.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
```

Using serve-static module the problem is solved.